gaek/ndb_json.py

```diff
@@ -68,13 +68,26 @@ def encode_generator(obj):
 
 
 def encode_key(obj):
+  """Get the Entity from the ndb.Key for further encoding. Alias for encode_key_as_entity()."""
+  # NOTE(erichiggins): Potentially poor performance for Models w/ many KeyProperty properties.
+  # NOTE(ronufryk): Potentially can cause circular references and "RuntimeError: maximum recursion depth exceeded"
+  return encode_key_as_entity(obj)
```

Comment:
Rather than calling this function, here's a more simple way to do function aliases:

```
def encode_key_as_entity(obj):
   blah blah blah
   ...

# Alias for backwards-compatibility.
encode_key = encode_key_as_entity
```